nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/serialization/RunningComponentSetFilter.java

```diff
@@ -39,6 +41,7 @@ public class RunningComponentSetFilter implements ComponentSetFilter {
     private final Map<String, VersionedControllerService> controllerServices = new HashMap<>();
     private final Map<String, VersionedProcessor> processors = new HashMap<>();
     private final Map<String, VersionedReportingTask> reportingTasks = new HashMap<>();
+    private final Map<String, VersionedFlowRegistryClient> flowRegistryClients = new HashMap<>();
```

Comment:
I don't see any reason to keep track of the Flow Registry Clients here either. They are never used. Not sure that they need to be added to ComponentSetFilter at all, but I might be wrong. Don't see how they'd get used in that context yet.